### PR TITLE
Sustainability -> top management message page

### LINKED ISSUE #518: Sustainability -> top management message page
As discussed during our call today.

Please use the background variant to integrate the following hero banner: https://main--sunstar--sunstar-global.hlx.live/jp/sustainability/message
image

We also want to use that hero banner on the main sustainability page instead of the bellow one (because of the contrast ratio issue): https://main--sunstar--sunstar-global.hlx.live/jp/sustainability
image

Please make sure the mobile version is adapted as well.

Thanks a lot.

@sdmcraft

### PR DESCRIPTION
**Pull request by shrotia**
Added composite variant at HeroBanner block

Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after), along with a short summary of changes:

## Issue

Fixes #518 

## Changelog:

- Added composite variant at HeroBanner block
- Added Page Level style for custom look & feel of sustainability section

## Test URLs:
- Original: https://www.sunstar.com/sustainability
- Before: https://main--sunstar--hlxsites.hlx.live/_drafts/shroti/sustainability
- After: https://issue-518--sunstar--hlxsites.hlx.live/_drafts/shroti/sustainability
- After: https://issue-518--sunstar--hlxsites.hlx.live/_drafts/shroti/message

### CODE REVIEW

**Comment by shrotia** [comment]
@ehrro Please look at the Hero Section on the following pages

https://issue-518--sunstar--hlxsites.hlx.live/_drafts/shroti/message
https://issue-518--sunstar--hlxsites.hlx.live/_drafts/shroti/sustainability

To maintain consistency and uniformity, I have followed the experience of [0] on both these pages.

There was minor difference in terms of how text was aligning at Mobile view on both of these pages (snapshot attached of the original site), I have kept it uniformed as per [0] for both of these. 

image

image

let me know if you see any gap, I will reiterate accordingly. Also, please have a look at the authoring part of the same [1].

[0] https://www.sunstar.com/jp/sustainability/message/ 
[1] https://issue-518--sunstar--hlxsites.hlx.live/_drafts/shroti/message?view-doc-source=true

File: styles/pages/sustainability.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+/* Desktop & Tablet */
+@media (min-width: 62rem) {
+    body.sustainability main .block.composite.two-column .composite-item-right img {
+        float: right;
+    }
+
+}
+
+/* only Mobile */
+@media (max-width: 62rem) {
```
Comment (sdmcraft) [inline styles/pages/sustainability.css:49]: Usually we are  avoiding `max-*` queries in this project.

**Review by sdmcraft — APPROVED** [review]
LGTM. Minor comments.

File: styles/pages/sustainability.css (hunk 1 above)
Comment (shrotia) [reply styles/pages/sustainability.css:49]: sure, let me remove that

**Comment by shrotia** [comment]
> Also , we should ensure that PSI check passes before we merge.


It is intermittently failing with an attached reason for Mobile.
Screenshot 2023-12-11 at 1 56 46 PM

Screenshot 2023-12-11 at 1 56 36 PM

Screenshot 2023-12-11 at 1 52 50 PM

Is this a known issue?

**Review by ehrro — CHANGES_REQUESTED** [review]
Hello @shrotia 

On desktop:
1. There is a margin on the h3 and the composition elements down there:
image
image

On mobile:
1. The padding should be 1 rem:
image

2. The text is cut
image

Is it possible to use this image for mobile:
su-chairman-mobile

Thanks.

**Comment by shrotia** [comment]
> There is a margin on the h3 and the composition elements down there:

Is it fine if I just go with the margin-bottom `1rem`?  Or you suggest to remove the margin altogether. 

As original pages have margin-bottom `2rem` & `0.5rem` respectively. 

image
image

**Comment by shrotia** [comment]
> The padding should be 1 rem:

Done.

> Is it possible to use this image for mobile:

I have replaced the Mobile image with the one attached in this PR.

**Comment by ehrro** [comment]
It's perfect now. Thank you @shrotia

**Review by ehrro — APPROVED** [review]

**Pull request merged**